subsys/debug/gpio_frontend/gpio_frontend.c

```diff
@@ -0,0 +1,663 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <logging/log_frontend.h>
+#include <hal/nrf_gpio.h>
+
+#if (CONFIG_LOG_FRONTEND_PORT_NUMBER == 0)
+	#define LOG_FRONTEND_GPIO_REG_PTR NRF_P0
+	#define BASE_ADDRESS 0x50000000
```

Comment:
Is it possible to get this value from the MDK?

Reply:
NRF_P0_BASE